packages/blocks/timer/.gitignore

```diff
@@ -0,0 +1,2 @@
+dist
```

Comment:
We don’t need `.gitignore` in new blocks. It’s a part of the template for those who want to start a new repo. Our top-level `.gitignore` covers both paths.